sdk/boms/azure-sdk-bom/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.24 (2024-05-29)
+
+### Dependency Updates
+
+- Updated Azure SDK dependency versions to the latest releases.
```

Comment:
it would be nice if this text included some of the interesting highlights from the SDK CHANGELOGS it is consuming